tsconfig.json

```diff
@@ -0,0 +1,33 @@
+{
+  "compilerOptions": {
+    "target": "ES2017",
+    "allowImportingTsExtensions": true,
+    "lib": ["dom", "dom.iterable", "esnext"],
+    "allowJs": true,
+    "skipLibCheck": true,
+    "strict": true,
+    "noEmit": true,
+    "esModuleInterop": true,
+    "module": "esnext",
+    "moduleResolution": "bundler",
+    "resolveJsonModule": true,
+    "isolatedModules": true,
+    "jsx": "preserve",
+    "incremental": true,
+    "plugins": [
+      {
+        "name": "next"
+      }
+    ],
+    "paths": {
+      "~/*": ["./*"]
+    }
```

Comment:
Don't use this. Use subpath imports.

Reply:
Subpath imports isn't support out of the box by Webpack/next.js 😭

Reply:
Next still uses webpack? 😵

Reply:
> Next still uses webpack? 😵

Yeah there are trying to create a rust webpack (turbopack) for them instead of using rspack F#ck